sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample05_SessionProcessor.md

```diff
@@ -1,6 +1,7 @@
 ## Using the Session Processor
 
-This sample demonstrates how to use the session processor. The session processor offers automatic completion of processed session messages, automatic session lock renewal, and concurrent execution of user specified event handlers.
+This sample demonstrates how to use the session processor. The session processor offers automatic completion of 
+processed session messages, automatic session lock renewal, and concurrent execution of user specified event handlers.
```

Comment:
Hmm, I didn't realize that. So the fact the fact that it will be wrapped automatically, means we should not add line breaks ourselves, or that we don't have to?

Reply:
We shouldn't line break, correct. We can do so in code blocks, obviously. But normal text doesn't usually have line breaks for markdown.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/13550